api/v1alpha1/nonadminrestore_types.go

```diff
@@ -61,6 +61,12 @@ type NonAdminRestoreStatus struct {
 	// phase is a simple one high-level summary of the lifecycle of an NonAdminRestore.
 	Phase NonAdminPhase `json:"phase,omitempty"`
 
+	// logsPath is path to backup logs in the storage location
```

Comment:
typo backup?

Reply:
Nope

Reply:
restore logs are saved in same path as backup logs?

Reply:
Good catch

Reply:
Ok typo it is